src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -29,41 +29,39 @@ interface RoomModalProps {
 }
 
 export const RoomModal: React.FC<RoomModalProps> = ({ show, onHide, room }) => {
-  const { user, profile } = useUser();
+  const { user } = useUser();
+  const userId = user?.uid;
 
   const venues = useSelector(orderedVenuesSelector);
   const venueEvents = useSelector(venueEventsSelector) ?? [];
 
-  const roomTitle = room?.title;
-  const userLastSeenIn = profile?.lastSeenIn;
-
-  const { recentRoomUsers } = useRecentRoomUsers(roomTitle);
-
   const roomVenue = useMemo(() => {
     if (!room) return undefined;
 
     return venues?.find((venue) => room.url.endsWith(`/${venue.id}`));
   }, [room, venues]);
 
-  const venueName = roomVenue?.name;
+  // const getRoomTitle = (room, venues) => {
+  //   const roomVenue = venues?.find((venue) => room.url.endsWith(`/${venue.id}`));
+  // }
 
-  // TODO: @debt refactor this to use openRoomWithCounting
```

Comment:
This is probably still a relevant note. When I implemented the refactor that cleaned up/standardised most of the usages to a single helper function in `src/utils/useLocationUpdateEffect.ts` there were a few places that diverged enough from the 'standard pattern' that I didn't have time to figure out how to make it work, which I labelled with this `@debt` sort of thing.

The intent was that in a future cleanup, we would centralise the remaining logic into a helper function/hook in the same sort of way, and none of this 'custom' stuff would live in any of the individual components.

I know this a WIP PR, but I just wanted to capture my intent/plan around this, so that you can make sure to fit that into the changes you're making here.